app/app.go

```diff
@@ -650,3 +606,44 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 
 	return paramsKeeper
 }
+
+// RegisterUpgradeHandlers returns upgrade handlers
+func (app *App) RegisterUpgradeHandlers(cfg module.Configurator) {
+	app.UpgradeKeeper.SetUpgradeHandler(v1UpgradeName, func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		minCommissionRate := sdk.NewDecWithPrec(5, 2)
+
+		fmt.Println("\nvm:\n", vm)
```

Comment:
whoops, we should delete this line

Reply:
Done in recent [committ](52b5ef4d0f8a8ce848b556c1e98ad518ca19f8d2)